go.mod

```diff
@@ -2,18 +2,18 @@ module github.com/charmbracelet/lipgloss
 
 retract v0.7.0 // v0.7.0 introduces a bug that causes some apps to freeze.
 
-go 1.17
+go 1.18
 
 require (
 	github.com/mattn/go-runewidth v0.0.15
 	github.com/muesli/reflow v0.3.0
 	github.com/muesli/termenv v0.15.2
+	github.com/rivo/uniseg v0.4.4
```

Comment:
Should we use [`v0.4.6`](https://github.com/rivo/uniseg/releases/tag/v0.4.6)

Reply:
Upgraded.